lib/emoji.js

```diff
@@ -1,5 +1,6 @@
 /*jslint node: true*/
 require('string.prototype.codepointat');
+var toArray = require('lodash/toArray');
```

Comment:
So the issue is with some complex emojis. 
let's take an example. 
```
var str = '👩‍❤️‍💋‍👩';

/* using lodash's toArray */
toArray(str);  //  outputs [ '👩‍❤️‍💋‍👩' ] (length: 1)

/* using Array.from() */
Array.from(str);  //  outputs: [ '👩', '‍', '❤', '️', '‍', '💋', '‍', '👩' ] (length : 8)

/* using string split() */
str.split('');  //  outputs [ '�', '�', '‍', '❤', '️', '‍', '�', '�', '‍', '�', '�' ] (length : 11)
```
I searched quite a bit to achieve this without lodash but no luck 👎 .